apps/web/pages/[...slug].vue

```diff
@@ -40,8 +39,8 @@ const { data: categoryTree } = useCategoryTree();
 const { buildCategoryLanguagePath } = useLocalization();
 
 const handleQueryUpdate = async () => {
-  await fetchProducts(getFacetsFromURL());
-  checkFiltersInURL();
+  await fetchProducts(getFacetsFromURL()).then(() => checkFiltersInURL());
```

Comment:
what is the difference? shouldn't it work the same with the await and next line?

Reply:
👍